File: backend/src/main/java/com/festago/dto/MemberTicketFestivalResponse.java
package com.festago.dto;

import com.festago.domain.Festival;

public record MemberTicketFestivalResponse(Long id, String name, String thumbnail) {

    public static MemberTicketFestivalResponse from(Festival festival) {
        return new MemberTicketFestivalResponse(festival.getId(), festival.getName(), festival.getThumbnail());
    }
}

File: android/festago/app/src/main/java/com/festago/festago/data/dto/MemberTicketFestivalResponse.kt
package com.festago.festago.data.dto

import com.festago.festago.model.MemberTicketFestival
import kotlinx.serialization.Serializable

@Serializable
data class MemberTicketFestivalResponse(
    val id: Int,
    val name: String,
    val thumbnail: String,
) {
    fun toDomain(): MemberTicketFestival = MemberTicketFestival(
        id = id,
        name = name,
        thumbnail = thumbnail,
    )
}
